### PR TITLE
feat: put blog posts into sitemap and update dependencies

### PR DESCRIPTION
**Pull request by feedm3**
- All dependencies are now up-to-date, except for `react-hook-form`. Will fix this in a separate PR
- I added the blog posts to the `sitemap.xml` to make them visible for Google

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ module.exports = {
     {
       resolve: `gatsby-plugin-sitemap`,
       options: {
-        exclude: ['/imprint', '/data-privacy', '/blog/**'],
+        excludes: ['/imprint', '/data-privacy'],
```
Comment (georgiee) [inline gatsby-config.js:135]: oh did that never work as we used the wrong keyword?

**Review by georgiee — CHANGES_REQUESTED** [review]
Thanks for thinking about this 🙌
1. The local http://localhost:8000/sitemap.xml doesn't show the blog posts? Wrong place to check?
2. The tests are failing, probably because it's outdated only?

File: gatsby-config.js (hunk 1 above)
Comment (feedm3) [reply gatsby-config.js:135]: - the sitemap plugin api changed due to the dependency update
- the sitemap only gets created on build, not with the dev server. you can see the sitemap here https://satellytescommain-choredependencies.gtsb.io/sitemap.xml
- will have a look for the tests

Comment (feedm3) [reply gatsby-config.js:135]: test is now fixed, pipeline is green. thanks for the review :)

Comment (georgiee) [reply gatsby-config.js:135]: I can't see the blog posts in this file though? (same as on localhost by the way).
I thought it's about the individual posts?
Screenshot 2021-06-15 at 16 27 09

Comment (feedm3) [reply gatsby-config.js:135]: Oh, nice catch, didn't notice it.

The new sitemap plugin has a new path for the sitemap, it's `sitemap/sitemap-index.xml`. I updated the URL to our `robots.txt`. Maybe you can also do a double check in the Google Search Console and update the path to the sitemap there. Full path should be `https://satellytes.com/sitemap/sitemap-index.xml` after the merge.

- https://satellytescommain-choredependencies.gtsb.io/sitemap/sitemap-index.xml
- https://satellytescommain-choredependencies.gtsb.io/sitemap/sitemap-0.xml

The root sitemap isn't generated anymore (missing on my local environment). It's probably on the gatsby deployment due to file caching.

Comment (georgiee) [reply gatsby-config.js:135]: clarified and answered 👍

**Review by georgiee — APPROVED** [review]
Let's merge that beauty of housekeeping

**Pull request merged**